helper/validation.go

```diff
@@ -410,3 +410,20 @@ func isValidVersion(value string) (v int64, ok bool) {
 	}
 	return 0, false
 }
+
+func FloatAtLeastAndLessThan(min, maxExclusive float64) schema.SchemaValidateFunc {
```

Comment:
@jerome-leanspace if you have a better idea than introducing this new function, let me know :)

Reply:
I don't :/